src/components/input/SelectNext.tsx

```diff
@@ -46,15 +46,19 @@ function SelectOption<T>({
   const selected = !disabled && currentValue === value;
 
   return (
-    <Button
-      variant="custom"
-      classes={classnames(
-        'w-full ring-inset rounded-none !p-0',
-        'border-t first:border-t-0 bg-transparent',
+    <li
+      className={classnames(
+        'w-full ring-inset focus:ring outline-none rounded-none',
+        'border-t first:border-t-0 transition-colors whitespace-nowrap',
```

Comment:
Moving for `Button` to `li` allows for less style overrides to be needed, and `focus:ring` to be used instead of `focus-visible-ring`

Reply:
One other style that came for free with buttons was setting `cursor: pointer` (or the equivalent TW style, that will need to be added here too).